### PR TITLE
[Term Entry] PyTorch Tensor Operations: .softmax()

### PR DESCRIPTION
**Pull request by karishma-battina**
### Description
Introduce a new entry on the `.softmax()` term under PyTorch. The entry should go in a new file under `docs/content/pytorch/concepts/tensor-operations/terms/softmax/softmax.md.`

### Issue Solved
Closes https://github.com/Codecademy/docs/issues/8140

### Type of Change
- New Term for Existing Concept Entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by dakshdeepHERE — APPROVED** [review]
LGTM 🚀
